mangaki/mangaki/admin.py

```diff
@@ -9,81 +9,40 @@ class StaffInline(admin.TabularInline):
     model = Staff
     fields = ('role', 'artist')
 
-class AnimeAdmin(admin.ModelAdmin):
+class WorkAdmin(admin.ModelAdmin):
     search_fields = ('id', 'title')
     list_display = ('id', 'title', 'nsfw')
-    list_filter = ('nsfw',)
-    actions = ['make_nsfw', 'make_sfw']
-    inlines = [StaffInline]
-    readonly_fields = (
-        'deprecated_director',
-        'deprecated_composer',
-        'deprecated_author',
-        'deprecated_genre',
-        'deprecated_origin',
-        'deprecated_nb_episodes',
-        'deprecated_anime_type',
-        'deprecated_anidb_aid',
-        'deprecated_editor',
-        'deprecated_studio',
-    )
-
-    def make_nsfw(self, request, queryset):
-        rows_updated = queryset.update(nsfw=True)
-        if rows_updated == 1:
-            message_bit = "1 anime est"
-        else:
-            message_bit = "%s animes sont" % rows_updated
-        self.message_user(request, "%s désormais NSFW." % message_bit)
-    make_nsfw.short_description = "Rendre NSFW les animes sélectionnés"
-
-    def make_sfw(self, request, queryset):
-        rows_updated = queryset.update(nsfw=False)
-        if rows_updated == 1:
-            message_bit = "1 anime n'est"
-        else:
-            message_bit = "%s animes ne sont" % rows_updated
-        self.message_user(request, "%s désormais plus NSFW." % message_bit)
-    make_sfw.short_description = "Rendre SFW les animes sélectionnés"
-
-
-class MangaAdmin(admin.ModelAdmin):
-    search_fields = ('id', 'title')
-    list_display = ('id', 'title', 'nsfw')
-    list_filter = ('nsfw',)
+    list_filter = ('category', 'nsfw',)
     actions = ['make_nsfw', 'make_sfw', 'merge']
     inlines = [StaffInline]
     readonly_fields = (
-        'deprecated_mangaka',
-        'deprecated_writer',
-        'deprecated_genre',
-        'deprecated_origin',
-        'deprecated_vo_title',
-        'deprecated_manga_type',
-        'deprecated_editor',
+        'sum_ratings',
+        'nb_ratings',
+        'nb_likes',
+        'nb_dislikes',
+        'controversy',
     )
 
     def make_nsfw(self, request, queryset):
         rows_updated = queryset.update(nsfw=True)
         if rows_updated == 1:
-            message_bit = "1 manga est"
+            message_bit = "1 œuvre est"
```

Comment:
Peut-être serait-t-il pertinent d'employer de l'i18n pour la gestion des pluriels ?
C'est un peu "ridicule" de le faire à la main, on va devoir le nettoyer inévitablement :/.

(à voir dans une PR probablement)